addons/binding/org.openhab.binding.autelis/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,568 @@
+		<label>Pump 1</label>
+		<description>Pump 1</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="pump-pump2">
+		<item-type>String</item-type>
+		<label>Pump 2</label>
+		<description>Pump 2</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="pump-pump3">
+		<item-type>String</item-type>
+		<label>Pump 3</label>
+		<description>Pump 3</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="pump-pump4">
+		<item-type>String</item-type>
+		<label>Pump 4</label>
+		<description>Pump 4</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="pump-pump5">
+		<item-type>String</item-type>
+		<label>Pump 5</label>
+		<description>Pump 5</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="pump-pump6">
+		<item-type>String</item-type>
+		<label>Pump 6</label>
+		<description>Pump 6</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="pump-pump7">
+		<item-type>String</item-type>
+		<label>Pump 7</label>
+		<description>Pump 7</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="pump-pump8">
+		<item-type>String</item-type>
+		<label>Pump 8</label>
+		<description>Pump 8</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="chlor-chloren">
+		<item-type>Number</item-type>
+		<label>Chlorine</label>
+		<description>Chlorine</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="chlor-poolsp">
+		<item-type>Number</item-type>
+		<label>Chlorine Pool Setpoint</label>
+		<description>Chlorine Pool Setpoint</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="chlor-spasp">
+		<item-type>Number</item-type>
+		<label>Chlorine Spa Setpoint</label>
+		<description>Chlorine Sap Setpoint</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="chlor-salt">
+		<item-type>Number</item-type>
+		<label>Chlorine Salt Level</label>
+		<description>Chlorine Salt Level</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="chlor-super">
+		<item-type>Number</item-type>
+		<label>Chlorine Super Level</label>
+		<description>Chlorine Super Level</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="chlor-chlorerr">
+		<item-type>Number</item-type>
+		<label>Chlorine Error</label>
+		<description>Chlorine Error value</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="chlor-chlorname">
+		<item-type>String</item-type>
+		<label>Chlorine Model Name</label>
+		<description>Chlorine Model Name</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="lightscmd">
+		<item-type>String</item-type>
+		<label>Send Lighting command</label>
```

Comment:
Is there any possible shorter label? Note that this is usually shown in the widget as the main text.